pkg/networkservice/common/connect/server.go

```diff
@@ -1,5 +1,3 @@
-// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Can we remove previous copyright holders if the file is reworked? 🤔

Reply:
Fixed :)